packages/monitors/src/OptimisticOracleContractMonitor.js

```diff
@@ -52,6 +52,11 @@ class OptimisticOracleContractMonitor {
           return Object.values(overrides).every((param) => Object.keys(this.logger.levels).includes(param));
         },
       },
+      optimisticOracleUIBaseUrl: {
+        // Base URL for the Optimistic Oracle UI.
+        value: "https://placeholder.umaproject.org/",
```

Comment:
do we not know the URL now? would it not be `"https://optimisticoracle.umaproject.org/"` or `"https://oo.umaproject.org/"`

Reply:
We don't know the URL. I asked @Tulun and @daywiss and they said it hasn't been decided yet. We can update this when we have a prod url.